docs/resources/file.md.erb

```diff
@@ -61,6 +61,10 @@ an owner:
 
     it { should be_executable.by('owner') }
 
+any user:
```

Comment:
Can we change this (and the others) to "any user other than the owner or members of the file's group"? "any user" is actually incorrect. If the file is mode `004`, the owner and members of the group still won't be able to read it even though the "others" have read bit enabled.

Suggestions for a shorter string than what I provided are also welcome. :)

Reply:
Good catch, couldn't come up with a shorter string so I used yours. :)